src/app/containers/create-movie/displayError.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const DisplayError = ({ error }) => {
```

Comment:
We should consider returning the jsx directly instead of using curly braces and return statement. It is redundant

Reply:
solved at [6c63366](https://github.com/OctaFloare/flitnext-2024/pull/12/commits/6c63366ce437096d199fa5b5aafd3880fbd91fc9)